### ISSUE TITLE
Computing jacobians is extremely slow

### ISSUE THREAD

**Issue by bluestyle97** [zhujiapeng/resefa#5]
Hi, I find the process of computing jacobians is extremely slow when testing with a 512x512 pretrained stylegan2 model, is this normal?

**Comment by zhujiapeng** [comment]
Yes, it is normal. It will take about one hour when using a 1024x1024 model on 2080 Ti GPU. StyleGAN3 will take a much longer time. We will release the code to speed up the process of computing the jacobian in the future.
